python/pyarrow/_dataset.pyx

```diff
@@ -240,14 +240,22 @@ cdef class Dataset(_Weakrefable):
         columns : list of str, default None
             The columns to project. This can be a list of column names to
             include (order and duplicates will be preserved), or a dictionary
-            with {new_column_name: expression} values for more advanced
+            with {{new_column_name: expression}} values for more advanced
```

Comment:
did you mean to double-up the brackets here?

Reply:
This was present for similar docstrings in other scanner implementations, so I might have mistakenly used here as well.